js/pro/binance.js

```diff
@@ -64,6 +64,9 @@ module.exports = class binance extends binanceRest {
                 'watchTicker': {
                     'name': 'ticker', // ticker = 1000ms L1+OHLCV, bookTicker = real-time L1
                 },
+                'watchTickers': {
+                    'name': '!ticker', // !ticker or !miniTicker
```

Comment:
@sc0Vu this is tricky, because it's translated to "not ticker" in python, we need to make a string concatenation.